cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -580,14 +553,36 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
                     verifyNotificationStatus(requests, notificationSpy, 'Moving');
                 });
 
-                it('shows a notification when undo moving', function() {
+                it('show a notification when undo moving', function() {
```

Comment:
I'd recommend use proper naming convention for tests. https://jasmine.github.io/2.0/introduction.html#section-Grouping_Related_Specs_with_<code>describe</code>